frontend/src/components/SearchLozenge/SearchLozenge.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './SearchLozenge.css';
+
+const SearchLozenge = (props) => {
+  let { searchTerm } = props;
+  if (!searchTerm) {
+    searchTerm = '';
+  }
```

Comment:
Maybe `searchTerm = searchTerm || '';` in place of this if condition?

Reply:
Could also use [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html) to specify the default. (That would be my preference).

Reply:
Everytime I add propTypes to set the default value, it seems to break my code.